src/NCDialog.cc

```diff
@@ -1083,6 +1084,10 @@ void NCDialog::processInput( int timeout_millisec )
 
 		break;
 
+	    case KEY_F( 13 ):
```

Comment:
Please add a comment that says that this is the equivalent to Shift-F1 for certain $TERM types (linux? xterm?).